cmd/notation-azure-kv/main.go

```diff
@@ -34,3 +42,22 @@ func main() {
 		os.Exit(1)
 	}
 }
+
+func help() {
+	fmt.Printf(`notation-azure-kv - Notation - Notary V2 Azure KV plugin
+
+Usage:
+  notation-azure-kv <command>
+
+Version:
+  %s
+
+Commands:
+  get-plugin-metadata  Get plugin metadata
+  generate-signature   Sign artifacts with keys in Azure Key Vault
+  describe-key         Azure key description
```

Comment:
Should we order the commands in alphabetical order?

Reply:
Ordered.